addons/rose/addon/components/rose/filter-tags/index.js

```diff
@@ -0,0 +1,81 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+
+export default class RoseFilterTagsIndexComponent extends Component {
+  // =services
+
+  @service router;
+
+  // =attributes
+  get tags() {
+    const tags = [];
+
+    for (const key in this.args.tags) {
+      if (Array.isArray(this.args.tags[key])) {
+        const names = this.args.tags[key].map((item) => item.name);
```

Comment:
Some tags will have no name so we should default to ID here too (we made a change to use `name` instead of `displayName` in our item options)

Reply:
When we build out the @tags param, we are using `displayName()` which will already handle that for us. I felt this was more simple and removes the need to default to the id in the component. Thoughts?

Reply:
I'm not sure if you saw but that's changing in lis's [PR](https://github.com/hashicorp/boundary-ui/pull/2053/files). We can't actually assume the `displayName` property for cases where we don't pass in a model

Reply:
Fixed.